andela/codepirates-ah-frontend
# #169194624 Update article

User should be able to edit their articles

#### What does this PR do?

  Add article update feature

#### Tasks to be accomplished

  - Pre-fill update form with the corresponding target form details
  - Update article field values in-place
  - Display response from the backend

#### How to manually test this feature

  - Enter http://localhost:8080/updatearticle/{slug}, where {slug} is the slug of an existing article
  - See the edit page preload the article
  - Go ahead and edit any fields and publish

### Related pivotal tracker stories
  [WIP #169194624](https://www.pivotaltracker.com/story/show/169194624)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/36167006/68109597-a1688380-fef3-11e9-8096-9e0759f421db.png)

#### Questions:
